cmd/serverless-init/tag/tag.go

```diff
@@ -62,11 +62,15 @@ func GetBaseTagsMapWithMetadata(metadata map[string]string, versionMode string)
 	return tagsMap
 }
 
-// WithoutContainerID creates a new tag map without the `container_id` tag
-func WithoutContainerID(tags map[string]string) map[string]string {
+// WithoutHihCardinalityTags creates a new tag map without high cardinality tags we use on traces
+func WithoutHighCardinalityTags(tags map[string]string) map[string]string {
 	newTags := make(map[string]string, len(tags))
 	for k, v := range tags {
-		if k != "container_id" {
+		if k != "container_id" &&
+			k != "gcr.container_id" &&
+			k != "gcrfx.container_id" &&
+			k != "replica_name" &&
+			k != "aca.replica.name" {
```

Comment:
Is there a more efficient approach for checking if a tag is in the exclude list?

Reply:
I checked for the same reason, but go doesn't have a filter reduce method.